### ISSUE TITLE
Swiping sometimes stops showing partial page

### ISSUE THREAD

**Issue by mvoong** [stefanceriu/SCPageViewController#13]
In the example app, it is possible for the scroll view to stop in the wrong place. You can reproduce this by swiping rapidly with multiple fingers, sometimes in opposite directions. After it gets to this state, swiping works as normal, until it happens again.

![animation stuck](https://cloud.githubusercontent.com/assets/571634/8909907/d378aa4a-347b-11e5-9980-5ca840b20009.png)

I looked at the code to see if there was anything obvious going wrong but I didn't find anything.

This happens on an iPhone 6 running iOS 8.4 and the Simulator running 8.4 and 8.1.

**Comment by stefanceriu** [comment]
Right, I think I have an idea where that's coming from. I'm going to try to fix it asap.

**Comment by stefanceriu** [comment]
I'm having a real bad time trying to reproduce this constantly but it looks like it only happens when interacting with the picker view.
Can you do me a favor and try to reproduce it after removing the picker view from the hierarchy ?

**Comment by mvoong** [comment]
Sorry for the delay, been tied up with other work.

Ah, yes, the problem completely goes away when there isn't a nested scroll view. It seems like during scrolling of the page view controller, if you interact with an inner scroll view, it cancels scrolling. Setting scrollEnabled to NO on the inner scroll views fixes this.

**Comment by stefanceriu** [comment]
Okay, cool. I'll keep looking into it, maybe there's a way around this limitation.

**Comment by mitchellporter** [comment]
Currently experiencing this on a project as well, but we don't have any inner scroll views. Sometimes there is a little space on the left and you can see the previous page, other times it looks similar to the image above with a large space on the right.

**Comment by gralexey** [comment]
Same, without inner scroll views

**Comment by AlMalKii07** [comment]
**__**
